public/chrome/btn_logout/btn_logout.js

```diff
@@ -29,23 +29,39 @@
  * permissions and limitations under the License.
  */
 
+import React from 'react';
```

Comment:
Why do we need to import React?

Reply:
Yes, we use React rendering framework here.